install_vendored_dynamic_framework/after/execution_output.txt

```diff
@@ -1004,6 +1004,7 @@ Generating Pods project
   - Adding resources to Pods project
   - Linking headers
   - Installing targets
+    - Installing target `Realm` iOS 8.0
```

Comment:
same, now all targets properly get a message that are being installed for visibility.